src/components/TextField/TextField.stories.mdx

```diff
@@ -21,41 +23,60 @@ import { FIGMA_URL } from '../../utils/common';
   }}
 />
 
-# TextField
+<SectionHeader title={'TextField'} />
 
-A universal TextField component that can be used as the <input/> in any form.
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
 
-## Usage
-
-```js
-import { TextField } from '@orfium/ictinus';
+## Overview
 
-<TextField label={'label'} />;
-```
+A universal TextField component which provides a wide range of features such as statuses, inline suffixes
+such as icons and components, and a multi-input functionality with the help of the Chip component.
 
 ## Props
 
 <Props of={TextField} />
 
-# Text Field with Label and Placeholder
+## Usage
+
+<UsageGuidelines
+  guidelines={[
+    'Do not display TextFields in the same row unless they are related (e.g. “First Name” / “Last Name”)',
+    'Explicitly mark optional fields as such',
+    'When possible, try to choose input width that is relevant to the content being entered into the input field',
+    'Avoid truncating field label and input as much as possible',
+  ]}
+/>
+
+<Tip>
+  When the user needs to enter more than one individual values at a given field that are not
+  selected from a predefined list, MultiTextField is more suitable. <br />
+  (scroll down for more details)
+</Tip>
+
+## Variants
+
+### TextField with Label and Placeholder
 
 Regular TextField with label options. Label that will float, with or without Placeholder
 
 <Preview>
-  <Story name="Text Field with Label and Placeholder">
+  <Story name="TextField with Label and Placeholder">
     <Stack>
       <TextField label={'Label'} />
       <TextField label={'Label'} placeholder={'Placeholder'} />
     </Stack>
   </Story>
 </Preview>
 
-# Text Field Icons
+### TextField with Icons
 
-TextField with icon options right and left of the text
+TextField with icon options right of the text, either as external image or one of the available ictinus Icons.
 
 <Preview>
-  <Story name="Text Field with Icons">
+  <Story name="TextField with Icons">
     <Stack>
       <TextField
```

Comment:
one last thing i forgot is that each icons on the left or right must be `aria-hidden={true}`